### PR TITLE
test: add integration test for opentsdb put api

### PR DESCRIPTION
**Pull request by zouxiang1993**
## Rationale
Add tests for #1037

### CODE REVIEW

File: integration_tests/opentsdb/test-put.py [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+    # empty tag
+    execute_put_then_assert_fail("""
+{
+    "metric": "sys.cpu.nice",
+    "timestamp": {ts},
+    "value": 18,
+    "tags": {
+    }
+}
+    """.replace('{ts}', str(ts)))
```
Comment (jiacai2050) [inline integration_tests/opentsdb/test-put.py:92]:
```
print('Hello {name}!This is {program}.'.format(name='foo',program='john'))
```
format already support name replacement.

Comment (zouxiang1993) [reply integration_tests/opentsdb/test-put.py:92]: The template is a json string which contains many '{' and '}' characters, so it's not convenient to use `format`

File: integration_tests/opentsdb/test-put.py [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+    r = requests.post('{}/sql'.format(api_root), json={'query': sql}, headers=headers)
+    return r
+
+def drop_table(table_name):
+    sql = """
+DROP TABLE {}    
+    """.format(table_name)
+    r = execute_sql(sql)
+    # 200 for success, 500 for table not found.
+    assert r.status_code == 200 or r.status_code == 500, r.text
```
[the remark this answers is not in the document]
Comment (zouxiang1993) [reply integration_tests/opentsdb/test-put.py]: fixed.

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**